bakery/src/pipelines/cops.js

```diff
@@ -149,19 +191,19 @@ const pipeline = (env) => {
     }
   ]
 
-  const gitPdfJob = {
-    name: 'PDF (git)',
-    build_log_retention: {
-      days: buildLogRetentionDays
-    },
-    plan: [
-      { get: 'output-producer-git-pdf', trigger: true, version: 'every' },
-      reportToOutputProducerGitPdf(Status.ASSIGNED, {
+  let resource
+  let report
+
+  resource = 'output-producer-git-pdf'
+  report = reportToOutputProducer(resource)
```

Comment:
Minor: Perhaps somewhat DRY-er if this `report` is passed as input to `wrapGenericCorgiJob` in each of these cases in lieu of L100 above (and the same can probably be done to `runWithStatusCheck` in lieu of L82). Or, at least I think that could work 🤷‍♂️ .

Reply:
Hmmm... an interesting suggestion.  `wrapGenericCorgiJob` will never use any reporter other than the one generated from `resource`, and it will need `resource` anyways, right? I agree it's dry-er, but it also seems weird to call a function `someFunction(x, y)` where the y=f(x) in all cases for something other than performance reasons (i.e. f(x) is expensive and should not be recalculated). I'm genuinely a bit curious about this, since I'm always interested in writing cleaner code, but I'm not actually sure what's cleaner here. What do you think?

Reply:
Yeah -- I wrote / deleted / re-pasted the comment in my review pass as well because I debated this question myself. What probably nudged me to keep the comment in was the thought that if we (for whatever reason) ever change the signature of `reportToOutputProducer` the fact that a setup reporter gets passed through might be cleaner. This may be more of a premature line of thought, though 😅, as that might never even happen .

Reply:
I see. I'm gonna err on the side of "when we come to it" on this one, I think.